docs/vgclog_examples.md

```diff
@@ -0,0 +1,595 @@
+
+### Scavenge partial GC cycle
+
+The following example is taken from a `gencon` log. The output is broken down into sections with supporting text to explain the GC processing that is taking place.
+
+To search for a scavenge partial GC cycle, you can search for the `type` attribute value `scavenge` in `cycle-start` and `cycle-end` elements, or search for the `<af>` element that logs the allocation failure trigger.
+
+By default, the `gencon` partial GC cycle runs by using a single *stop-the-world* (STW) pause. The cycle performs only 1 operation,  a *scavenge* operation, which runs only on the *nursery* area. The cycle consists of a single GC increment, which is labelled using the elements shown in the following table:
+
+|GC Operation | GC increment | STW or concurrent| XML element of GC increment          | Details                                                                   |
+|-------------|--------------|------------------|--------------------------------------|---------------------------------------------------------------------------|
+|Scavenge     |Single        | STW              | `<gc-start>`, `<gc-end>` |Contains detailed information about objects being copied and weak roots processing|
+
+The scavenge partial GC cycle follows a general structure in the verbose GC log as shown. The lines are indented to help illustrate the flow and some elements are omitted for clarity:
+
+```xml
+
+<exclusive-start/>          (STW Pause starts)
+
+<af-start/>                 (allocation failure trigger recorded)
+
+<cycle-start/>              (scavenge cycle starts)
+
+<gc-start>                  (scavenge cycle increment starts)
+
+<mem-info>                  (memory status before operation)
+
+<mem></mem>                 (status of different types of memory)
+
+</mem-info>
+
+</gc-start>
+
+<allocation-stats/>         (snapshot of application threads status taken before cycle started)
+
+<gc-op> “scavenge"</gc-op>  (scavenge operation completed)
+
+<gc-end>                    (scavenge cycle increment ends)
+
+<mem-info>                  (memory status after operation)
+
+<mem></mem>                 (status of different types of memory)
+
+</mem-info>
+
+</gc-end> 
+
+</cycle-end>                (scavenge cycle ends)
+
+<allocation-satisfied/>     (required allocation has been achieved)
+
+<af-end/>
+
+<exclusive-end>             (STW for scavenge cycle ends)
+...
+
+```
+
+The first activity in the cycle is recorded by an `<exclusive-start>` element, which indicates the start of the STW pause. Application threads are halted to give
+the garbage collector exclusive access to the heap:
+
+```xml
+<!-- Start of gencon scavenge partial GC cycle example -->
+
+<exclusive-start id="12392" timestamp="2020-10-18T13:35:45.000" intervalms="406.180">
+  <response-info timems="0.070" idlems="0.070" threads="0" lastid="00000000013D6900" lastname="LargeThreadPool-thread-68" />
+</exclusive-start>
```

Comment:
This deserves a sentence or even a paragraph (if so not necessarily here), where we explain that stopping mutator threads is asynchronous mechanism.... (I can explain in person), and what are the meanings of those times. Does not have to be a part of this PR.

Reply:
Adding to a new github issue